rpc/grpc/client/src/lib.rs

```diff
@@ -99,28 +106,39 @@ impl GrpcClient {
             counters,
         )
         .await?;
+        let client_id = u64::from_le_bytes(rand::random::<[u8; 8]>());
```

Comment:
what's the worst-case scenario if this id gets duplicated?

Reply:
No impact at all.

This id is only used in direct mode for building `subscriptions` and is actually ignored by `start_notify` and `stop_notify`. It can be removed and replaced by a default 0 at all occurrences.